pubtools/_content_gateway/utils.py

```diff
@@ -255,3 +226,56 @@ def sort_items(items):
     sorted_items.extend(product_delete)
 
     return sorted_items
+
+
+def formate_cgw_items(items):
```

Comment:
formate?
https://en.wikipedia.org/wiki/Formate

Reply:
It should have been "format". Will update.